ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -6004,5 +5997,68 @@
     "CaseFieldID": "previousProceedingListCTSC",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanList",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanListLA",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanListCTSC",
```

Comment:
looks like admin, gatekeeper and judges can't upload application document, only LA can upload them. 
Does that mean the CTSC list is always empty?

Reply:
Not sure if the new management document UI flow allows them to do so. Worth to keep them here for consistency. Perhaps I will implement this constraint in the new manage document flow.